src/main/java/org/sagebionetworks/bridge/udd/helper/FileHelper.java

```diff
@@ -15,11 +22,38 @@
  */
 @Component
 public class FileHelper {
```

Comment:
Seems like a good candidate eventually for a shared library. DateUtils could be a similar component, again for mocking.